modules/migrations/base/retry_downloader.go

```diff
@@ -48,6 +48,9 @@ func (d *RetryDownloader) GetRepoInfo() (*Repository, error) {
 		if repo, err = d.Downloader.GetRepoInfo(); err == nil {
 			return repo, nil
 		}
+		if IsErrNotSupported(err) {
```

Comment:
Maybe you mean
```go
if !IsErrNotSupported(err)
```
?

Reply:
alog goes like this:
1. if sucess return
2. if IsErrNotSupported(err) return err since it doesnt mather how mouch you retry ..
3. retry

Reply:
@lunny ping